src/index.js

```diff
@@ -3,7 +3,7 @@ import bodyParser from 'body-parser';
 import logger from 'morgan';
 import cors from 'cors';
 import swaggerUI from 'swagger-ui-express';
-import swaggerDocument from '../public/api-docs/swagger.json';
+import swaggerDocument from '../public/api-docs/swagger';
```

Comment:
move the `swagger` folder to `src/docs/swagger`

Reply:
Do you mean I should move the `./public/api-docs` contents to `src/docs/swagger`?

Reply:
yeah.